### PR TITLE
FF module update and description modernization.

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Right now a lot of modules reference Jaeger despite the fact they also work with Xenonauten. Some of their descriptions are misleading with the weight section in particular having no standard. So I changed them while working on another PR, but because atomization is good I have split this part up. This is also a good place for discussion of the changes. 

As for the actual module changes:
Valk has .05 more slowdown, because even numbers are nice and it's pretty good.
And finally the friendly fire module now has laser armor due to their new prevalence throughout marines ranks. That's not all though, it also has a new name so it matches the other modules names after norse gods
Höðr is the blind son of Odin and Frigg, he is most known for having been tricked into killing the invincible Baldr with a mistletoe arrow given to him by Loki (that's right, a divine friendly fire accident).

## Why It's Good For The Game

Better descriptions make clear where a module goes, what it does and if it needs anything (Mimir mask and helmet module), and how it will affect you (slowdown standards, ex: 0.2 = "Will impact mobility"). 
Valk slowdown nerf is mostly just to get a nice even number, I can round down if maintainers want. Besides, it's not like .05 will ruin the incredible usefulness of the valk.
Seeing as to how prevalent laser weapons are between marines, a good argument can be made to grant the friendly fire module laser armor. Xenomorphs have no laser attacks so this shouldn't be any issue.
And giving it a cool name just brings it on the same level as other neat modules such as Hlin or Mimir.

## Changelog
:cl:
add: standardizes, modernizes, and improves all the module descriptions.
balance: FF armor module now has laser armor, and a cool new name!
/:cl:

### CODE REVIEW

**Comment by revelation-8468** [comment]
hehe vor(e) module

**Comment by Arctrooper8** [comment]
> hehe vor(e) module

God damn it

**Review by TiviPlus — CHANGES_REQUESTED** [review]
A) Surt rebalance is in the same PR as desc and name changes??
B) do not rename the ballistic armor typepath for loadout reasons
C) Puck some other name not Vor choose like Hodr

**Comment by Arctrooper8** [comment]
> A) Surt rebalance is in the same PR as desc and name changes?? B) do not rename the ballistic armor typepath for loadout reasons C) Puck some other name not Vor choose like Hodr

Will do all of those things! Surt will be seperate PR, will get a new god (Stupid vore joke) and I will not change the typepath.

**Comment by Arctrooper8** [comment]
> A) Surt rebalance is in the same PR as desc and name changes?? B) do not rename the ballistic armor typepath for loadout reasons C) Puck some other name not Vor choose like Hodr

All complete! Here is the new PR: https://github.com/tgstation/TerraGov-Marine-Corps/pull/9745

File: code/modules/clothing/modular_armor/attachments/modules.dm [hunk 1]
```diff
@@ -110,12 +110,12 @@
...
 /obj/item/armor_module/module/mimir_environment_protection
 	name = "\improper Mark 2 Mimir Environmental Resistance System"
-	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. When activated, this system provides substantial resistance to environmental hazards, such as gases, biological and radiological exposure. This newer version provides a large amount of resistance to acid. Best paired with the Mimir Environmental Helmet System and a gas mask. Will impact mobility." // Add the toggable thing to the description when you are done, okay? ~XS300
+	desc = "Designed for mounting on modular armor. This newer model provides great resistance to acid, biological, and radiological attacks. Pairing this with a Mimir helmet module and mask will make the user impervious to xeno gas clouds. Will impact mobility."
 	icon = 'icons/mob/modular/modular_armor_modules.dmi'
 	icon_state = "mod_biohazard"
 	item_state = "mod_biohazard_a"
 	soft_armor = list("bio" = 40, "rad" = 50, "acid" = 30)
-	slowdown = 0.2
+	slowdown = 0.2 //Something this useful needs a downside.
```
Comment (TiviPlus) [inline code/modules/clothing/modular_armor/attachments/modules.dm:118]:
```suggestion
	slowdown = 0.2
```

**Review by TiviPlus — APPROVED** [review]
rest ok

**Comment by Arctrooper8** [comment]
> rest ok

Committed your suggestion, ready to merge!

**Pull request merged**